cmd/commands/cli/command_identities.go

```diff
@@ -164,8 +164,8 @@ func (c *cliApp) getIdentity(actionArgs []string) (err error) {
 	if err != nil {
 		return err
 	}
-	clio.Info("Registration Status:", identityStatus.RegistrationStatus)
-	clio.Info("Channel address:", identityStatus.ChannelAddress)
+	clio.Info("Registration Status: ", identityStatus.RegistrationStatus)
```

Comment:
Can we fix this for unlock as well: https://github.com/mysteriumnetwork/node/blob/2c1e2bf05fd137aaf3209164f78d394507f32281/cmd/commands/cli/command_identities.go#L215